hydrolib/core/io/friction/models.py

```diff
@@ -0,0 +1,145 @@
+import logging
+from pathlib import Path
+from typing import List, Literal, Optional, Set, Union
+
+from pydantic import Field
+from pydantic.class_validators import root_validator, validator
+
+from hydrolib.core.io.ini.models import INIBasedModel, INIGeneral, INIModel
+from hydrolib.core.io.ini.util import (
+    get_enum_validator,
+    get_from_subclass_defaults,
+    get_split_string_on_delimiter_validator,
+)
+from hydrolib.core.utils import str_is_empty_or_none
+
+from ..ini.util import make_list_validator
+
+logger = logging.getLogger(__name__)
+
```

Comment:
I think we need some simple tests for the friction classes, to make sure the instances can be properly initialized with kwargs, +/- optional parameters.

Reply:
See 98a1e47.